CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ Bug fixes:
 - Fix bug in `apply` with `axis=1` where the given user defined function returns nested 
   data in the geometry column (#2959)
 
+Deprecations and compatibility notes:
+ - ``GeometryArray.geom_almost_equals()`` has been deprecated and  
```

Comment:
```suggestion
 - ``geom_almost_equals()`` methods have been deprecated and  
```
This would be probably more precise as it is deprecated on both GeometryArray and GeoPandasBase.